deploy/docker/Dockerfile

```diff
@@ -3,19 +3,26 @@ FROM fluent/fluentd:v1.11.1-debian-1.0 AS builder
 # Use root account to use apt
 USER root
 
-# Build dependencies
-RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev libsnappy-dev" \
- && apt-get update \
- && apt-get install -y --no-install-recommends $buildDeps
-
-COPY gems/fluent-plugin*.gem ./
+# Dependencies
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
```

Comment:
```suggestion
 && apt-get install --yes --no-install-recommends \
```

Reply:
what about other `apt install`s in this file? I'd just leave this as is

Reply:
I've added suggestion for the other `apt-get install` as well.

Reply:
Aa a rule of thumb I like to specify all the flags in their full name so that the flag is less cryptic.

Reply:
done